packages/PHPStanStaticTypeMapper/TypeMapper/NeverTypeMapper.php

```diff
@@ -24,8 +24,11 @@ public function getNodeClass(): string
     /**
      * @param NeverType $type
      */
-    public function mapToPHPStanPhpDocTypeNode(Type $type): TypeNode
+    public function mapToPHPStanPhpDocTypeNode(Type $type, ?string $kind = null): TypeNode
     {
+        if ($kind === 'return') {
```

Comment:
Here the `TypeKind::*` should be used

Reply:
there were a few other mappers which also used the plain string, which inspired me for this line.

I changed all of them to `TypeKind` now.

interessting is the following place:
https://github.com/rectorphp/rector-src/blob/0b717e1df41c952169335877a45c753e1e3a00c1/packages/PHPStanStaticTypeMapper/TypeMapper/StaticTypeMapper.php#L49